apps/root/src/pages/history/components/historyTable/index.tsx

```diff
@@ -209,29 +209,35 @@ const formatTokenElement = (txEvent: TransactionEvent): React.ReactElement => {
     case TransactionEventTypes.EARN_INCREASE:
       return (
         <>
-          {txEvent.data.asset.icon}
+          {txEvent.data.depositToken.icon}
           <StyledCellContainer direction="column">
             <StyledBodySmallRegularTypo2 noWrap maxWidth={'6ch'}>
-              {txEvent.data.asset.symbol || '-'}
+              {txEvent.data.depositToken.symbol || '-'}
             </StyledBodySmallRegularTypo2>
             <StyledBodySmallLabelTypography noWrap maxWidth={'12ch'}>
-              {txEvent.data.asset.name || '-'}
+              {txEvent.data.depositToken.name || '-'}
             </StyledBodySmallLabelTypography>
           </StyledCellContainer>
         </>
       );
     case TransactionEventTypes.EARN_WITHDRAW:
-      const tokens = txEvent.data.withdrawn.map((withdrawn) => withdrawn.token);
-      const assetToken = tokens.find((token) => token.address === txEvent.data.assetAddress);
-      const isWithdrawingRewards = tokens.some((token) => token.address !== txEvent.data.assetAddress);
+      const withdrawnAsset = txEvent.data.withdrawn[0];
```

Comment:
this actually seems a bit random that it will always be the first, what happens if they later change that?

Reply:
I'm asking for the assetAddress anyway so we can get the transactionValue in usd. So I think this will change in the next commit. Anyway it's a convention as mentioned before, but I agree it's a little sensitive for future changes